backend/src/modules/auth/commands/handlers/register.handler.ts

```
@@ -0,0 +1,61 @@
+					dob: registerDto.dateOfBirth,
+					bio: registerDto.bio,
+					nickName: registerDto.nickName,
+				},
+			},
+		};
+
+		const newAccount: RegisterResponseDto = await this.authRepository.createAccount({
+			account: accountData,
+		});
+
+		delete newAccount.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)